api/user/schema.py

```diff
@@ -33,6 +33,8 @@ class Arguments:
 
     def mutate(self, info, **kwargs):
         user = UserModel(**kwargs)
+        if not verify_email(user.email):
+            raise GraphQLError("Can\'t add a none Andela email")
```

Comment:
Please use Non-Andelan

Reply:
@cwizard2011 this has been fixed.

Reply:
Hey @dnuwa, great work. Since a user is created by an Admin. I feel like we can do more about this error message. It could be `You can not register a user with a non-Andelan email`. It is just a suggestion.

Reply:
@nakatuddesuzan your message is good and very descriptive but I think its way too long. So, I think to have it short and equally descriptive is good